lib/MetaCPAN/Document/File/Set.pm

```diff
@@ -499,28 +499,47 @@ sub autocomplete_using_suggester {
         }
         );
 
-    my @docs
-        = map { $_->{text} } @{ $suggestions->{documentation}[0]{options} };
+    my %docs;
+    for my $d ( @{ $suggestions->{documentation}[0]{options} } ) {
+        next if exists $docs{ $d->{text} };
+        $docs{ $d->{text} } = $d->{score};
+    }
+
+    my @docs_uniq = sort { $docs{$b} <=> $docs{$a} } keys %docs;
```

Comment:
Out of curiosity, why do these need to be sorted if it's being passed to Es in a query?